src/MongoDB/Driver/Manager.cpp

```diff
@@ -522,6 +534,136 @@ static bool hippo_mongo_driver_manager_apply_ssl_opts(mongoc_client_t *client, c
 	return 1;
 }
 
+void command_started(const mongoc_apm_command_started_t *event)
+{
+	ObjectData *obj_context = (ObjectData*) mongoc_apm_command_started_get_context(event);
+	Class *cls = obj_context->getVMClass();
+	static Class *c_event;
+	Func  *m   = cls->lookupMethod(s_MongoDBDriverManager_dispatchCommandStarted.get());
+	Variant    result;
+	MongoDBDriverManagerData* data = Native::data<MongoDBDriverManagerData>(obj_context);
+
+	Array subscribers = obj_context->o_get(s_MongoDBDriverManager_subscribers, false, s_MongoDriverManager_className).toArray();
+
+	if (subscribers.size() < 1) {
+		return;
+	}
+
+	c_event = Unit::lookupClass(s_MongoDriverMonitoringCommandStartedEvent_className.get());
+	Object o_event = Object{c_event};
+	o_event->o_set(s_MongoDriverMonitoringCommandEvent_commandName, Variant(mongoc_apm_command_started_get_command_name(event)), s_MongoDriverMonitoringCommandEvent_className);
+	o_event->o_set(s_MongoDriverMonitoringCommandEvent_server, Variant(hippo_mongo_driver_server_create_from_id(data->m_client, mongoc_apm_command_started_get_server_id(event))), s_MongoDriverMonitoringCommandEvent_className);
+	o_event->o_set(s_MongoDriverMonitoringCommandEvent_operationId, Variant(mongoc_apm_command_started_get_operation_id(event)), s_MongoDriverMonitoringCommandEvent_className);
+	o_event->o_set(s_MongoDriverMonitoringCommandEvent_requestId, Variant(mongoc_apm_command_started_get_request_id(event)), s_MongoDriverMonitoringCommandEvent_className);
+	o_event->o_set(s_MongoDriverMonitoringCommandStartedEvent_databaseName, Variant(mongoc_apm_command_started_get_database_name(event)), s_MongoDriverMonitoringCommandStartedEvent_className);
```

Comment:
o_set() requires a Variant to be passed. Earlier, it would auto convert to a Variant, but that got removed some time ago.